### ISSUE TITLE
Kyocera Hydro unlisteds

### ISSUE THREAD

**Issue by Rudxain** [0x192/universal-android-debloater#809]
**Your phone model:** C5170

**Packages:**

```
com.android.camera
com.android.launcher
com.android.wallpaper
com.google.android.apps.genie.geniewidget
com.google.android.gallery3d
com.google.android.inputmethod.latin.dictionarypack
com.kyocera.diagservice
com.kyocera.dm.service
com.kyocera.ecomode
com.kyocera.hiddendebug
com.kyocera.MipErrorDialog
com.kyocera.monkeyscript
com.kyocera.omadm
com.kyocera.outputstorage
com.kyocera.section7.vvmcounter
com.kyocera.sysupdater
com.kyocera.wallpaper
com.locationlabs.v3client
com.sprint.dsa
com.sprint.internal
com.sprint.w.prefact
```

- [ ] **I removed all those packages on my phone**
      Some were removed. Some are unsafe to remove

### com.google.android.gallery3d

**List**: `AOSP`
**Removal**: `Advanced`

> Built-in Gallery app. The ID could be "recycled" by OEMs for their own gallery implementations

### com.kyocera.ecomode

**List**: `OEM`
**Removal**: `Unsafe`

> Custom power-saving mode

## Other info

I rooted (then unrooted) that phone some years ago, so some unlisted packs may be missing, because I completely uninstalled them

**Comment by AnonymousWP** [comment]
Can you add some documentation/description etc. to it as well?

**Comment by Rudxain** [comment]
> Can you add some documentation/description etc. to it as well?

Yes. I currently don't have access to the device, but I remember some packs
